core/pom.xml

```diff
@@ -98,6 +98,10 @@
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-jboss-marshalling</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>pnc-common</artifactId>
```

Comment:
```suggestion
      <artifactId>common</artifactId>
```

Reply:
It might build, but it still won't work as there is a datatype issue between `int` and `long` so we are going to be getting the wrong ids for ids greater than a certain value, but I can fix this.